src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/RequestForPriceEstimate/RequestForPriceEstimateScreen.tsx

```diff
@@ -117,3 +132,19 @@ export const RequestForPriceEstimateScreen: React.FC<RequestForPriceEstimateScre
     </FormikProvider>
   )
 }
+
+const tracks = {
+  trackSentRequestPriceEstimate: (
```

Comment:
Because tracks just return the event and don't send it, we usually use the following naming convention for tracks:

```suggestion
  sentRequestPriceEstimate: (
```

Reply:
right, thanks for spotting this